init.cpp:139:34: error: 'strcmp' was not declared in this scope
init.cpp:196:12: error: 'exit' was not declared in this scope
Index: src/init.cpp
--- src/init.cpp.orig
+++ src/init.cpp
@@ -26,8 +26,9 @@
 #include "scorescreen.h"
 #include <fstream>
 #include <iostream>
+#include <string.h>
+#include <stdlib.h>
 using namespace std; 
-
 
 int cheat       = 0;
 int zappy;
